src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-form/rbd-snapshot-form-modal.component.ts

```diff
@@ -153,4 +159,16 @@ export class RbdSnapshotFormModalComponent extends BaseModal implements OnInit {
       this.createAction();
     }
   }
+
+  // "." and "/" not accepted for snapshot name
+  specialCharacterValidator(): ValidatorFn {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const value = control.value?.trim();
+      const onlyDotsAndSlashes = /^[./]+$/.test(value);
+      if (onlyDotsAndSlashes) {
+        return { invalidCharacters: { value: control.value } };
+      }
+      return null;
+    };
+  }
```

Comment:
Maybe we can move this to the `cdvalidators.ts` where all the validators lay.

Instead of writing a comment, is better to have a more descriptive name
```suggestion
  validateNoDotOrSlash(): ValidatorFn {
    return (control: AbstractControl): ValidationErrors | null => {
      const value = control.value?.trim();
      const onlyDotsAndSlashes = /^[./]+$/.test(value);
      if (onlyDotsAndSlashes) {
        return { invalidCharacters: { value: control.value } };
      }
      return null;
    };
  }
```